tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -5243,6 +5243,131 @@ TEST (nnstreamer_capi_singleshot, invoke_dynamic_fail_n)
   g_free (test_model);
 }
 
+TEST (nnstreamer_capi_property, test_mux_00)
+{
+  ml_pipeline_h handle;
+  ml_pipeline_element_h elem_h;
+
+  gchar *pipeline;
+  gchar *ret_mode = NULL;
+  int status;
+
+  pipeline = g_strdup("videotestsrc ! video/x-raw,format=RGB,width=640,height=480 ! videorate max-rate=1 ! " \
+    "tensor_converter ! tensor_mux name=mux ! tensor_demux ! tensor_sink");
+
+  status = ml_pipeline_construct (pipeline, NULL, NULL, &handle);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+
+  status = ml_pipeline_element_get_handle (handle, "mux", &elem_h);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+
+  status = ml_pipeline_element_set_property(elem_h, "sync_mode", "slowest", NULL);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+
+  status = ml_pipeline_element_set_property (elem_h, "no_property", "nosync", NULL);
```

Comment:
Can you separate negative case? Also please consider to add testcase with multiple peoperties.

Reply:
Right. I need to reorganize and add more test cases.
These test cases just show the name of API and its usage.

If this implementation is OK, then I will do that.